docs/workflows/lambda.md

```diff
@@ -6,123 +6,64 @@ hide_title: true
 
 # Develop Lambda Functions
 
-## Develop a new Lambda
+## Develop a new Cumulus Lambda
 
-To develop a new Lambda from a sample, create a new folder in `cumulus/tasks/` and run `npm init`:
+AWS provides great getting started guide for building Lambdas in the [developer guide](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html).
 
-```bash
-  $ cd ../cumulus/tasks
-  $ mkdir new-lambda
-  $ cd new-lambda
-  $ npm init
-```
-
-Or copy an existing Lambda function to customize:
+Cumulus currently supports the following environments for Cumulus Message Adapter enabled functions:
 
-```bash
-  $ cd ../cumulus/tasks
-  $ cp discover-pdrs new-lambda
-```
+* [Node.js 8.10](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
+* [Java 8](https://docs.aws.amazon.com/lambda/latest/dg/java-programming-model.html)
+* [Python 2.7, 3.6](https://docs.aws.amazon.com/lambda/latest/dg/python-programming-model.html)
 
-Modify package.json:
+Addtionally you may chose to include any of the other languages AWS [supports](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) as a resource with reduced feature support.
```

Comment:
`reduced feature support` = no CMA integration?

Reply:
Yes.   In addition to losing the CMA optional features, due to the decision to tightly couple metrics exports to CMA functionality it's also required for that as well.

Reply:
Yeah, I guess that is explicitly stated when we say that we only support the CMA for the listed environments 👍